iothub/service/src/Authentication/Security/SharedAccessSignatureParser.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+
+namespace Microsoft.Azure.Devices
+{
+    internal sealed class SharedAccessSignatureParser
+    {
+        /// <summary>
+        /// Parses a shared access signature string representation into a <see cref="SharedAccessSignature"/>.
+        /// </summary>
+        /// <param name="iotHubName">The IoT hub name.</param>
+        /// <param name="rawToken">The string representation of the SAS token to parse.</param>
+        /// <returns>The <see cref="SharedAccessSignature"/> instance that represents the passed in raw token.</returns>
```

Comment:
<see> tag not needed. The type is already evident and linkable by being part of the API definition.
```suggestion
        /// <returns>The shared access signature instance that represents the passed in raw token.</returns>
```